### PR TITLE
Composition vs Inheritance

### PR DESCRIPTION
**Pull request by mihilbabin**
Added translation for `Composition vs Inheritance` section.

### CODE REVIEW

**Comment by mihilbabin** [comment]
Should be merged after #9.

File: content/docs/composition-vs-inheritance.md [hunk 1]
```diff
@@ -156,17 +156,17 @@ class SignUpDialog extends React.Component {
...
+У Facebook ми використовуємо React у тисячах компонентів і ми не знайшли жодного випадку використання в якому б ми могли порадити створення ієрархій наслідування компонентів.
 
-Props and composition give you all the flexibility you need to customize a component's look and behavior in an explicit and safe way. Remember that components may accept arbitrary props, including primitive values, React elements, or functions.
+Пропси і композиція дають вам всю необхідну гнучкість, щоб налаштувати вигляд і поведінку компонента в явний і безпечний спосіб. Пам'ятайте, що компоненти можуть приймати довільні пропси, включно з примітивами, React-елементами чи функціями.
 
-If you want to reuse non-UI functionality between components, we suggest extracting it into a separate JavaScript module. The components may import it and use that function, object, or a class, without extending it.
+Якщо ви хочете повторно використати функціональність котра не відноситься до інтерфейсу користувача, ми рекомендуємо витягнути її в окремий JavaScript модуль. Компоненти можуть імпортувати його і використувати функцію, об'єкт чи клас без наслідування.
```
Comment (ithompson4) [inline content/docs/composition-vs-inheritance.md]: `, котра` missing comma

Comment (ithompson4) [inline content/docs/composition-vs-inheritance.md]: `, котра` missing comma

File: content/docs/composition-vs-inheritance.md [hunk 2]
```diff
@@ -156,17 +156,17 @@ class SignUpDialog extends React.Component {
...
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/gwZbYa?editors=0010)
+[**Спробувати на CodePen**](https://codepen.io/gaearon/pen/gwZbYa?editors=0010)
 
-## So What About Inheritance? {#so-what-about-inheritance}
+## То що з приводу наслідування? {#so-what-about-inheritance}
 
-At Facebook, we use React in thousands of components, and we haven't found any use cases where we would recommend creating component inheritance hierarchies.
+У Facebook ми використовуємо React у тисячах компонентів і ми не знайшли жодного випадку використання в якому б ми могли порадити створення ієрархій наслідування компонентів.
```
Comment (ithompson4) [inline content/docs/composition-vs-inheritance.md]: missing comma `, в якому` or `, у якому`

File: content/docs/composition-vs-inheritance.md [hunk 3]
```diff
@@ -106,14 +106,14 @@ function WelcomeDialog() {
   return (
     <Dialog
       title="Welcome"
```
Comment (klymenkoo) [inline content/docs/composition-vs-inheritance.md]: Перекласти `Welcome`

**Comment by klymenkoo** [comment]
@mihilbabin looks good, thank you  for contribution :)

**Pull request merged**